src/Template/Element/plugin_class_error.ctp

```diff
@@ -29,5 +29,3 @@ else:
         '<a href="http://book.cakephp.org/3.0/en/plugins.html#autoloading-plugin-classes">Plugins - autoloading plugin classes</a>'
     );
 endif;
-
-?>
```

Comment:
Templates should keep the closing tags - as they are mainly HTML, and there might be some more HTML that wants to be added below at some point.